src/validation/__tests__/NoIntrospectionFieldsRule-test.js

```diff
@@ -0,0 +1,100 @@
+// @flow strict
+
+import { describe, it } from 'mocha';
+
+import { NoIntrospectionFieldsRule } from '../rules/NoIntrospectionFieldsRule';
+
+import { expectValidationErrors } from './harness';
+
+function expectErrors(queryStr) {
+  return expectValidationErrors(NoIntrospectionFieldsRule, queryStr);
```

Comment:
it's better to use `expectValidationRulesWithSchema` and have a short SDL snippet in this file than to depend on a huge schema.
I was planning to break "harness" into multiple small schemas and put them directly into test files so for new rules let's do it from the start.
BTW. Are you interested in working on breaking "harness" schema?

Reply:
I can do that. Maybe I could knock that out as a separate PR first and then rebase this.

Reply:
Actually, if you are interested in doing that work, we can merge this first and your later change all the rules in one decisive PR.

Reply:
That works too!